### PR TITLE
fix: Change response when no errors found

### PR DESCRIPTION
**Pull request by Bobbyperson**
The old message implies that an error occurred while reading the log. This is clearer.

### CODE REVIEW

File: reaper/cogs/log_reading.py [hunk 1]
```diff
@@ -432,17 +432,19 @@ async def on_message(self, message):
                     value="Please note that I am a bot and am still heavily being worked on. There is a chance that some or all of this information is incorrect, in which case I apologize.\nIf you still encounter issues after doing this, please send another log.",
                     inline=False,
                 )
-                await message.channel.send(embed=problem, reference=message, view=view)
+                await message.reply(embed=problem, reference=message, view=view)
```
Comment (GeckoEidechse) [inline reaper/cogs/log_reading.py:435]: The change to `reply` seems to break it something so I'm reverting it ^^"

Traceback (most recent call last):
  File "Reaper/.venv/lib/python3.12/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "Reaper/reaper/cogs/log_reading.py", line 435, in on_message
    await message.reply(embed=problem, reference=message, view=view)
  File "Reaper/.venv/lib/python3.12/site-packages/discord/message.py", line 1564, in reply
    return await self.channel.send(content, reference=self, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: discord.abc.Messageable.send() got multiple values for keyword argument 'reference'


```suggestion
                await message.channel.send(embed=problem, reference=message, view=view)
```

**Review by GeckoEidechse — COMMENTED** [review]
I'm not entirely sure if this solves the problem or just further hides as we only hit this part when we set `problem_found` to `true`  but then set nothing in the embed.

Which tbh should never happen in the first place ^^"

If there were no issues with the log, we'd never hit this part in the first place.

**Comment by Bobbyperson** [comment]
Wait is that actually what's happening? I need to look into this deeper

**Comment by Bobbyperson** [comment]
I think it has something to do with duplicate audio checking???? maybe????? I'm being held back by not knowing what the hell most of this code is doing, I might rewrite the audio aspect since it seems to be a bit dubious anyways.

**Comment by GeckoEidechse** [comment]
Tbh, might just be worth rewriting the log parsing first with a focus on regex or something as mentioned in #58 before attempting to fix the current spaghetti logic.

**Pull request closed without merging**